projects/lib/src/board/westernboard.h

```diff
@@ -201,7 +201,9 @@ class LIB_EXPORT WesternBoard : public Board
 		 * FEN string which succeed the en passant field.
 		 */
 		virtual QString vFenIncludeString(FenNotation notation) const;
-
```

Comment:
```
/*!
 * Set the castling target for \a side towards castling side \a cside to square index \a square.
 */
```

Reply:
Protected and public methods should have a comment above their declarations.

Reply:
Thank you, I have add the comment above the declaration.